app/stac_api/migrations/0034_merge_0032_alter_asset_file_0033_auto_20240704_1157.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 5.0.6 on 2024-07-09 13:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('stac_api', '0032_alter_asset_file'),
+        ('stac_api', '0033_auto_20240704_1157'),
+    ]
+
+    operations = []
```

Comment:
was this migration done automatically? Merging migrations has not always worked well in the past, but this is already a while ago, so it's probably reliable now.

Reply:
Yes this was created automatically because of the conflicts with the changes for the new s3 bucket. I am not aware of any migration issues, it seems to work so far.